docs/GlobalAPI.md

```diff
@@ -731,6 +731,60 @@ test.each`
 });
```

Comment:
should it work with `each` as well?

Reply:
I didn't plan to add this functionality to each because of the way I thought the users will be using this. If needed I can try to add it but it will be hard for me to manage because of the branching of behavior. 

Also the docs are currently mentioning all the variants of all the calls. We will need at least I think 16 descriptions of `failing` with `skip`, `only`, `concurrent.skip`, `concurrent.only`, `concurrent`, `concurrent.each.skip`....... I think if we were to add this we would need to rethink how the docs explain examples.